### PR TITLE
[ $1 ] Update README.md

### PR DESCRIPTION
**Pull request by nthypes**
closes #1

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Este é um comentário.
+*       @givebk-bot
```
Comment (mfrma-moura) [inline CODEOWNERS:2]: just a test

**Review by mfrma-moura — CHANGES_REQUESTED** [review]
just a test

**Comment by nthypes** [comment]
@givebk-bot !target $100

**Comment by nthypes** [comment]
@givebk-bot !pledge $0.20

**Pull request closed without merging**